### ISSUE TITLE
Trying to install a non-existent driver gives authorization error

### ISSUE THREAD

**Issue by smola** [bblfsh/bblfshd#207]
From https://github.com/src-d/engine/issues/104 (thanks @BFergerson):

> I accidentally typed the command `srcd parse drivers install list` instead of `srcd parse drivers list` and received error:

```
INFO[0000] installing driver                             language=list version=latest
ERRO[0000] unable to install version latest of list driver: rpc error: code = Unknown desc = can't install list driver: rpc error: code = Unknown desc = errors:
denied: requested access to the resource is denied
unauthorized: authentication required
```

> `authentication required` is an odd error for this mistaken command.

**Comment by dennwc** [comment]
This error comes from the Dockerhub. We can detect it and rewrite an error message, but if you try to download a private Docker image without proper credentials, an error will be "cannot find the driver" which is also misleading. But I guess this is not a common use case right now, so overriding an error makes sense.

**Comment by smola** [comment]
@dennwc We can also make it explicit in the error, something like `The driver does not exist or it is private`
or maybe `The driver does not exist` if it is `bblfsh` organization and `The driver does not exist or it is private` otherwise.

**Comment by bzz** [comment]
Fixed in #213
